azurelinuxagent/ga/exthandlers.py

```diff
@@ -873,42 +875,96 @@ def initialize(self):
     def enable(self):
         self.set_operation(WALAEventOperation.Enable)
         man = self.load_manifest()
+        handler_configuration = None
+        try:
+            handler_configuration = self.load_handler_configuration()
+        except ExtensionError as e:
+            self.logger.debug('Failed to load resource manifest file: {0}'.format(ustr(e)))
```

Comment:
should be warning or info

Reply:
Done; Made it to `info`, as most of the extensions might not have this file.